doc/metric-api.md

```diff
@@ -417,3 +434,98 @@ public class MD5ReportConsumer {
     }
 }
 ```
+
+### 7.PrometheusExports
+
+According to EaseAgent naming standard and calculation value rules, we have redefined Prometheus Exports.
```

Comment:
Will our documents include the standard and rules?

Reply:
This topic is our standards and rules